test/tests/functional/pbs_cpuset.py

```diff
@@ -142,7 +246,7 @@ def test_reliable_job_startup_on_cpuset(self):
 
         # Submit job2 requesting the released vnode, job runs
         j2 = Job(TEST_USER, {
-            ATTR_l + '.select': '1:ncpus=1:mem=2gb:vnode=%s' % vnodeB})
+            ATTR_l + '.select': '1:ncpus=4:mem=2gb:vnode=%s' % vnodeB})
```

Comment:
The hardwired 'ncpus=4', shouldn't this be according the actual number of 'resources_available.ncpus' in vnodeB?